radis/lbl/base.py

```diff
@@ -2017,49 +2017,44 @@ def _calc_Q(molecule, iso, state):
 
         if "id" in df1.columns:
```

Comment:
It is not possible to calculate HITRAN simulation because of this change

Reply:
sorry but I didn't really get what you wanted to do on this line so I hardly try to mess up

Reply:
Do you get AttributeError? 
If so, we could switch it to a self.warn(..., "PerformanceWarning") 

Id is the molecule number. For performance reasons it shouldn't be stored as a column, this takes unnecessary Memory space.  It's related to @gagan-aryan project. But until the column is removed and replaced with a simple attribute of the dataframe, having a warning should still allow you to compute HITRAn

Reply:
thank you for doing the correction :)